packages/flutter/lib/src/cupertino/text_selection_toolbar.dart

```diff
@@ -510,49 +526,126 @@ class _CupertinoTextSelectionToolbarContentState extends State<_CupertinoTextSel
   Widget build(BuildContext context) {
     return widget.toolbarBuilder(context, widget.anchor, widget.isAbove, FadeTransition(
       opacity: _controller,
-      child: _CupertinoTextSelectionToolbarItems(
-        page: _page,
-        backButton: CupertinoTextSelectionToolbarButton.text(
-          onPressed: _handlePreviousPage,
-          text: '◀',
-        ),
-        dividerWidth: 1.0 / MediaQuery.devicePixelRatioOf(context),
-        nextButton: CupertinoTextSelectionToolbarButton.text(
-          onPressed: _handleNextPage,
-          text: '▶',
-        ),
-        nextButtonDisabled: const CupertinoTextSelectionToolbarButton.text(
-          text: '▶',
+      child: AnimatedSize(
+        duration: _kToolbarTransitionDuration,
+        curve: Curves.decelerate,
+        child: GestureDetector(
+          onHorizontalDragEnd: _onHorizontalDragEnd,
+          child: _CupertinoTextSelectionToolbarItems(
+            key: _toolbarItemsKey,
+            page: _page,
+            backButton: CupertinoTextSelectionToolbarButton(
+              onPressed: _handlePreviousPage,
+              child: CustomPaint(
+                  painter: _CupertinoChevronPainter(
+                    color: _kToolbarTextColor.resolveFrom(context),
+                    iconSize: _kToolbarChevronSize,
+                    isLeft: true,
+                  ),
+                  size: const Size.fromWidth(_kToolbarChevronSize),
```

Comment:
This block is indented too far.

Reply:
I changed this block to a method because it was repeated for both chevrons, now it should have the correct indentation.